src/write.c

```diff
@@ -1674,64 +1674,13 @@ avifResult avifEncoderFinish(avifEncoder * encoder, avifRWData * output)
             avifRWStreamFinishBox(&s, dref);
             avifRWStreamFinishBox(&s, dinf);
 
+            // The boxes within the "stbl" box are ordered using the following recommendation in ISO/IEC 14496-12, Section 6.2.3:
+            // 4) It is recommended that the boxes within the Sample Table Box be in the following order: Sample Description
+            // (stsd), Time to Sample (stts), Sample to Chunk (stsc), Sample Size (stsz), Chunk Offset (stco).
+            //
+            // Any boxes not listed in the above line are placed in the end (after the "stco" box).
```

Comment:
Just wanted to confirm that stss is the only one, right?

Does Android expect stss to be after the other boxes listed above?

Reply:
> Just wanted to confirm that stss is the only one, right?

Yes, as of now `stss` is the only box not in the list. But the spec potentially allows for other boxes that aren't in the list. So i added the note just to make sure if we add any more boxes here, it has to be added after the `stco` box.

> 
> Does Android expect stss to be after the other boxes listed above?

The Android MP4 parser code is written in such a way that if the `stsz` box appears before the `stsd` box, then the parser fails to identify the image sequences properly. It is a bug and i will try to raise an issue with the Android team and/or try to fix it.